app/shared/resource-card/ResourceCardContainer.js

```diff
@@ -1,6 +1,211 @@
+import classnames from 'classnames';
+import round from 'lodash/round';
+import PropTypes from 'prop-types';
+import queryString from 'query-string';
+import React, { Component, Fragment } from 'react';
+import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import iconHome from 'hel-icons/dist/shapes/home.svg';
+import iconMapMarker from 'hel-icons/dist/shapes/map-marker.svg';
+import iconTicket from 'hel-icons/dist/shapes/ticket.svg';
+import iconUser from 'hel-icons/dist/shapes/user-o.svg';
+import iconHeart from 'hel-icons/dist/shapes/heart-o.svg';
 
+import { injectT } from 'i18n';
+import iconHeartFilled from 'assets/icons/heart-filled.svg';
+import UnpublishedLabel from 'shared/label/Unpublished';
+import iconMap from 'assets/icons/map.svg';
+import BackgroundImage from 'shared/background-image';
+import { getMainImage } from 'utils/imageUtils';
+import { getHourlyPrice, getResourcePageUrlComponents } from 'utils/resourceUtils';
+import ResourceAvailability from './label';
+import ResourceCardInfoCell from './info';
 import resourceCardSelector from './resourceCardSelector';
-import ResourceCard from './ResourceCard';
+import {
+  favoriteResource,
+  unfavoriteResource
+} from 'actions/resourceActions';
 
-export default connect(resourceCardSelector)(ResourceCard);
+class ResourceCard extends Component {
```

Comment:
Why did you rename the file to `ResourceCardContainer`?

Reply:
I think the file should still be `ResourceCard`.

Reply:
Just to keep it consistent. Other file have unconnected component in same place with container too.

Imo there is actually no need of extra container file for just like few lines, can revert if its not make sense

Reply:
@rBoost: if its just about file naming, not content -> Yes i can rename it to `ResourceCard`

Reply:
ah yeah, I hate that naming convention. The filename shouldn't change whether or not is it connected to redux.
We should stop using that convention, remove the `Container` from the filename.

Reply:
Agree.